src/sentry/workflow_engine/migration_helpers/alert_rule.py

```diff
@@ -288,7 +289,10 @@ def migrate_metric_data_conditions(
         type=threshold_type,
         condition_group=detector_data_condition_group,
     )
-
+    DataConditionAlertRuleTrigger.objects.create(
```

Comment:
I'm going to update the migration helper tests separately but I needed this to be here for the endpoint

Reply:
also updated the migration [here](https://github.com/getsentry/sentry/pull/85426/commits/4e38e7b295b45232d243a00bdf86d647100c1190)

Reply:
Updating it all here https://github.com/getsentry/sentry/pull/88968 as a follow up